.gitignore

```diff
@@ -43,3 +43,5 @@ default.profraw
 *.swp
 *.swo
 *.swn
+
+cmake-build-debug/
```

Comment:
Since this folder name is user provided and not automatically created, I don't think we should add it here.